cli-tests/tests/flow/pg-operator.spec.ts

```diff
@@ -44,21 +43,6 @@ test.describe('Everest CLI install', async () => {
         ]);
       });
     };
-    const clusterName = `test-${faker.number.int()}`;
-
-    await test.step('run everest install command', async () => {
-      const out = await cli.everestExecSkipWizard(
-        `install -v --operator.mongodb=false --operator.postgresql=true --operator.xtradb-cluster=false --namespaces=everest-operators`,
-      );
```

Comment:
No, the `-v` option is still there. I removed the test because I didn't think it was useful to test the same thing twice - once with pretty output and once with debug logs.. Especially now since Helm takes care of the main heavy lifting, there isn't much output in the verbose logs